bin/prepare_rt_kernel.sh

```diff
@@ -18,6 +19,9 @@ make mrproper
 cp /boot/config-`uname -r` .config
 
 make olddefconfig
+# Setting systemkeys to avoid; No rule to make target 'debian/certs/test-signing-certs.pem' build error
+scripts/config --set-str SYSTEM_TRUSTED_KEYS ""
```

Comment:
Where does this `scripts/config` come from? (I don't have it in my machine)

Reply:
its in the downloaded kernel folder ~/linux-*.*.*/scipts

Reply:
Oh, of course :sweat_smile: